docs/products/clickhouse/reference/supported-input-output-formats.md

```diff
@@ -16,6 +16,7 @@ exchange is possible with the following formats only:
 | **JSONEachRow**               | Example: `{"x":123,"y":"hello"}`                                                                                           |
 | **JSONStringsEachRow**        | Example: `{"x":"123","y":"hello"}`                                                                                         |
 | **MsgPack**                   | Example: `{\\xc4\\x05hello`. Libraries and documentation: [https://msgpack.org/](https://msgpack.org/)                     |
+| **Parquet**                   | Example: `{"x": [123], "y": [b"hello"]}`                                                                                   |
```

Comment:
```suggestion
| **Parquet**                   | Binary parquet format. Libraries and documentation: [https://parquet.apache.org/](https://parquet.apache.org/)             |
```

Like avro, we can't put a real example here because it's a big format, it would be way too long and not really helpful.